run.sh

```diff
@@ -0,0 +1,7 @@
+if [ ! -d savedata ]; then mkdir savedata; fi
+if [ ! -d vips_out ]; then mkdir vips_out; fi
+if [ ! -d vips_out/cache ]; then mkdir vips_out/cache; fi
+if [ ! -d vips_out/cache/shop ]; then mkdir vips_out/cache/shop; fi
+if [ ! -d vips_out/cache/items ]; then mkdir vips_out/cache/items; fi
+if [ ! -d vips_out/cache/cards ]; then mkdir vips_out/cache/cards; fi
```

Comment:
```suggestion
mkdir -p savedata vips_out/cache/shop vips_out/cache/items vips_out/cache/cards
```
Use the `-p` flag on `mkdir` which can recursively create files, and does not error if the directory already exists.